service/src/main/java/tds/config/web/endpoints/ConfigController.java

```diff
@@ -31,4 +33,17 @@ public ConfigController(ConfigService configService) {
 
         return ResponseEntity.ok(clientSystemFlag);
     }
+
+    @GetMapping(value = "{clientName}/messages/{context}/{messageKey}/{language}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseBody
+    ResponseEntity<String> getClientSystemMessage(@PathVariable final String clientName,
+                                                  @PathVariable final String language,
+                                                  @PathVariable final String context,
+                                                  @PathVariable final String messageKey,
+                                                  @RequestParam(required = false) final String subject,
+                                                  @RequestParam(required = false) final String grade) {
```

Comment:
I can't think of a better URL since language is required.  Unrelated, should we be using `languageCode` argument naming throughout?